.github/workflows/unit-test.yml

```diff
@@ -0,0 +1,29 @@
+name: Auto-Formatting
+
+on:
+  pull_request:
+    types: opened
+    jobs:
+      rebase:
+        - uses: actions/checkout@master
+        - name: Automatic Rebase
+          uses: cirrus-actions/rebase@master
+          env:
+            GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+on: [push]
+jobs:
```

Comment:
@DanVanAtta You might already have figured this out on your own, but each workflow (yaml file) can only have global triggers for all jobs.
Besides that you're defining the value for `on` 2 times, which is invalid yaml spec-wise.
However you are able to define all the triggers at once, and add an `if` key to the individual jobs to prevent them from running if the false trigger is the cause.

See https://help.github.com/en/actions/automating-your-workflow-with-github-actions/contexts-and-expression-syntax-for-github-actions#github-context and https://help.github.com/en/actions/automating-your-workflow-with-github-actions/workflow-syntax-for-github-actions#jobsjob_idif